msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages_eu.properties

```diff
@@ -907,3 +907,10 @@ cdfm_faq_forum_title=FAQ Forum
 cdfm_faq_forum_description=Forum for frequently asked questions
 cdfm_faq_topic_title=FAQ Topic
 cdfm_faq_topic_description=Topic for frequently asked questions
+
+#S2U-29
```

Comment:
@pablosanroman would you mind contributing these strings? Thanks!

Reply:
Hi @mpellicer , here are the lines

cdfm_faq_forum_description=Galdera ohikoentzako foroa
cdfm_faq_topic_title=FAQ gaia
cdfm_faq_topic_description=Galdera ohikoentzako gaia

Reply:
Thank you Pablo, strings updated.

I'm missing this one in EU

pvt_publish_to_faq=Publish to FAQ

Reply:
Ok @mpellicer !
pvt_publish_to_faq=Publikatu FAQ gisa